pkg/controller/restapi/restapi_test.go

```diff
@@ -1,8 +1,11 @@
 /*
-Copyright SecureKey Technologies Inc. All Rights Reserved.
```

Comment:
any reason to change the format ? Most of the files use this format.

Reply:
looks like my IDE is autofixing it.

Reply:
should revert this change.

Reply:
reverted